criu/seize.c

```diff
@@ -24,6 +24,46 @@
 #include "xmalloc.h"
 #include "util.h"
 
+char *task_comm_info(pid_t pid, char *comm, size_t size)
+{
+	int ret = 0;
+
+	if (!pr_quelled(LOG_INFO)) {
+		int saved_errno = errno;
+		char path[32];
+		int fd;
+
+		snprintf(path, sizeof(path), "/proc/%d/comm", pid);
+		fd = open(path, O_RDONLY);
+		if (fd >= 0) {
+			ssize_t n = read(fd, comm, size);
+			if (n > 0)
+				comm[n - 1] = '\0';
```

Comment:
Why do we override the last comm symbol?

Reply:
This is to replace `\n` with `\0`. https://github.com/torvalds/linux/blob/f3a2439f20d918930cc4ae8f76fe1c1afd26958f/fs/proc/base.c#L1702

Reply:
It isn't obvious. I suggest to add a comment to explain this.

Reply:
Sure, done in:
https://github.com/checkpoint-restore/criu/pull/2095